.drone.yml

```diff
@@ -21,6 +21,7 @@ pipeline:
     group: test
     image: ${IMAGE_PHP=fpfis/httpd-php-dev:5.6}
     commands:
+      - ./vendor/bin/grumphp
```

Comment:
The good way to add the code sniffer in the pipeline is 
>  grumphp:
>    group: test
>    image: ${IMAGE_PHP=fpfis/httpd-php-dev:5.6}
>    commands:
>      - ./vendor/bin/grumphp run

Reply:
Ops, my bad :(